eventstore/src/operations/mod.rs

```diff
@@ -28,15 +28,13 @@ crate::impl_options_trait!(OperationalOptions);
 pub struct StatsOptions {
     pub(crate) common_operation_options: crate::options::CommonOperationOptions,
     pub(crate) refresh_time: Duration,
-    pub(crate) use_metadata: bool,
```

Comment:
Does this mean we're releasing a new mayor version?

Reply:
We do but not for that specific reason/PR